server/controllers/services/putService.ts

```diff
@@ -0,0 +1,32 @@
+import { RequestHandler } from 'express';
+import { putServiceQuery } from '../../queries';
+import { servicesSchema } from '../../validation';
+import CustomizedError from '../../utils/error';
+
+const puttService: RequestHandler = async (req, res, next) => {
+  const { id } = req.params;
+  try {
+    const {
+      name, description, price, image, categoryId,
+    } = await servicesSchema.validate(req.body, { abortEarly: false });
+    const { rows, rowCount } = await putServiceQuery(
+      id as any,
```

Comment:
You are sending the `id` with the `req.body` and sending it to the database with the `id` that came from the `params` while you are not using it, remove it from the `req.body` and use the params only.

Reply:
I didn't use id from req.body, I get it from req.params
`const { id } = req.params;`